### DISCUSSION TITLE
How to limit image display size?

### DISCUSSION THREAD

**Discussion by FinnurHrafn** [OrchardCMS/OrchardCore#9984] [Q&A] [answered]
Hi all,
I have Orchard Core v1.00 default image running on my Windows 10 Docker Desktop using the Blog recipe/theme, everything on default settings. Everything works fine, but when I add an image to an Article content item, it extends outside the right side of the page in full resolution. Is there some easy way to limit the picture size to not exceed the containing div?

I like the extensive documentation but simple things like this are not easy to find.

**Comment by hishamco** [comment, marked as answer]
If you are using a custom theme you can easily fix this using CSS

**Reply by FinnurHrafn** [reply]
Yes, I knew that and I know I will  eventually need to make a custom theme to get the look I want. I was just curios to know if there was some simple way to do this without doing that.

I really like the documentation to help me understand the internals of OC but simple end user documentation is not easy to find.

**Reply by hishamco** [reply]
There's images sizes support, but I'm not sure if they are created after upload the original image, @deanmarcussen can confirm, but as I mentioned before custom CSS may solve your needs

**Reply by Skrypt** [reply]
Look for Image or Media Profiles feature.

**Reply by Skrypt** [reply]
![image](https://user-images.githubusercontent.com/3228637/127200022-edd8e2f8-eb1b-4331-9c60-cb550288c9f2.png)

**Reply by FinnurHrafn** [reply]
Hello @hishamco and @Skrypt, thanks for the quick responses. I should have spent more time reading the documentation :)
I am just using the default HTML editor, I was able to control the size with the width and height parameters. For some reason the rmode resize settings did not seem to have any effect. I am using the "mode Output" format, is that the appropriate format to use when using the HTML editor?

Anyway I need to spend more time studying the docs and experimenting.
